src/Symfony/Component/Form/Extension/Core/Type/TimeType.php

```diff
@@ -85,7 +87,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 });
             }
 
-            $builder->addViewTransformer(new DateTimeToStringTransformer($options['model_timezone'], $options['view_timezone'], $format));
+            $viewTransformer = new DateTimeToStringTransformer($options['model_timezone'], $options['view_timezone'], $format);
+
+            if (null !== $parseFormat) {
+                $property = new \ReflectionProperty($viewTransformer, 'parseFormat');
+                $property->setAccessible(true);
+                $property->setValue($viewTransformer, $parseFormat);
```

Comment:
Does that mean we're missing a constructor argument to DateTimeToStringTransformer maybe?

Reply:
I didn't add it on purpose as that felt like introducing a new extension point and thus a new feature to me. But I am fine with the change you did as well.